10/collection-meta.yaml

```diff
@@ -124,8 +124,6 @@ collections:
       - rainerleber
     repository: https://github.com/sap-linuxlab/community.sap_libs
   vmware.vmware_rest:
-    maintainers:
```

Comment:
Yeah, it's a problem that `vmware.vmware_rest` now has no named maintainers. Do you want to add yourself here as well?

Reply:
At the moment, I'm just helping out so: No, not really.

Reply:
Can we point to a Github team instead?  We would prefer to not have to maintain a static list of staff in multiple places, it's pretty much guaranteed to be out of date.
We use https://github.com/orgs/ansible-collections/teams/cloud

Reply:
I think the "we would prefer to not have to maintain a static list of staff in multiple places" a very good one.

What do you think @gotmax23?

Reply:
A team is fine IMO. As long as it is something we can ping on GitHub, it's fine :)

Reply:
It only works inside that org. That's enough to ping folks in issues inside that collection. It won't work to ping them from here though.